plugins/repository-gcs/build.gradle

```diff
@@ -61,7 +61,7 @@ dependencies {
   api 'com.google.api:api-common:1.8.1'
   api 'com.google.api:gax:1.54.0'
   api 'org.threeten:threetenbp:1.4.4'
-  api 'com.google.protobuf:protobuf-java-util:3.19.3'
+  api 'com.google.protobuf:protobuf-java-util:3.25.5'
```

Comment:
```suggestion
  api 'com.google.protobuf:protobuf-java-util:${versions.protobuf}'
```

Reply:
@owaiskazi19 I tried this yesterday but it failed:
```
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':plugins:repository-gcs:updateShas'.
> Could not resolve all files for configuration ':plugins:repository-gcs:runtimeClasspath'.
   > Could not find com.google.protobuf:protobuf-java-util:${versions.protobuf}.
     Required by:
         project :plugins:repository-gcs
```
There's probably some other fix to make it work but it's not this simple.